docs/auxil/admonition_inserter.py

```diff
@@ -265,12 +266,54 @@ def _create_returned_in(self) -> dict[type, str]:
                 except NotImplementedError as e:
                     raise NotImplementedError(
                         f"Error generating Sphinx 'Returned in' admonition "
-                        f"(admonition_inserter.py). {cls}, method {method}. "
+                        f"(admonition_inserter.py). {cls}, method {method_name}. "
                         f"Couldn't resolve type hint in return annotation {ret_annot}. {str(e)}"
                     )
 
         return self._generate_admonitions(methods_for_class, admonition_type="returned_in")
 
+    def _create_shortcuts(self) -> dict[collections.abc.Callable, str]:
+        """Creates a dictionary with 'Shortcuts' admonitions for Bot methods that
+        have shortcuts in other classes.
+        """
+
+        # pattern for looking for calls to Bot methods only
+        bot_method_pattern = re.compile(
+            r"""\s*  # any number of whitespaces
+            (?<=return\sawait\sself\.get_bot\(\)\.)  # lookbehind
+            .+  # the method name we are looking for
+            (?=\() # lookahead: closing bracket
```

Comment:
I'm not overly familiar with lookahead, so just double checking: If I were to type `return await self.get_bot().one().two()`, would this match `one` ore `one().two`?

Reply:
First of all, thanks for helping me spot a typo in the last line: it's actually an **opening** bracket. 

This regex will match `one`.

Reply:
The usual addition of `?` makes it non-greedy. This will match `one`:

```py
bot_method_pattern = re.compile(
    r"""\s*  # any number of whitespaces
    (?<=return\sawait\sself\.get_bot\(\)\.)  # lookbehind
    .+?  # NON-GREEDY
    (?=\() # lookahead: opening bracket before the args of the method start
    """,
    re.VERBOSE,
)
```

Reply:
Alternatively, I could be looking for only letters and underscores:

```py
bot_method_pattern = re.compile(
    r"""\s*  # any number of whitespaces
    (?<=return\sawait\sself\.get_bot\(\)\.)  # lookbehind
    \w+  # the method name we are looking for
    (?=\() # lookahead: opening bracket before the args of the method start
    """,
    re.VERBOSE,
)
```

Reply:
None of this is really relevant in our case because it's always a hanging open bracket after the `Bot` method, but maybe I should replace `.+` with `\w+` to make it match only letters and underscores.

Reply:
should be `[\w_]+`, then, right? Or are underscores included in `\w`?
Anyway, I'm fine with both :) the "non-greedy" part was what I wanted to ensure.

Reply:
Yes, funnily enough, underscores are included in `\w`.

Reply:
Done